diff -p -up grdesktop-0.23/src/rdpparse.c~ grdesktop-0.23/src/rdpparse.c
--- grdesktop-0.23/src/rdpparse.c~	2004-02-03 11:46:50.000000000 +0100
+++ grdesktop-0.23/src/rdpparse.c	2009-07-25 13:40:35.000000000 +0200
@@ -306,9 +306,9 @@ gchar *ext_geometry(const gchar *value, 
 	parts = g_strsplit(value, "x", 2);
 
 	if(typ == 'W')
-		return(g_strdup_printf(parts[0]));
+		return(g_strdup_printf("%s", parts[0]));
 	else
-		return(g_strdup_printf(parts[1]));
+		return(g_strdup_printf("%s", parts[1]));
 
 	if(parts != NULL)
 		g_strfreev(parts);
diff -p -up grdesktop-0.23/src/run.c~ grdesktop-0.23/src/run.c
--- grdesktop-0.23/src/run.c~	2004-03-30 15:17:53.000000000 +0200
+++ grdesktop-0.23/src/run.c	2009-07-25 13:39:23.000000000 +0200
@@ -110,7 +110,7 @@ void run_rdesktop() {
 		sshcmd = g_strconcat(sshcmd, g_strdup_printf("%s",
 			SHASH("sshhost")), NULL);
 		sshcmd = g_strconcat(sshcmd, g_strdup_printf(" \"%s\"", cmd), NULL);
-		g_warning(sshcmd);
+		g_warning("%s", sshcmd);
 		cmdline = g_strconcat(sshcmd, NULL);
 	} else {	/* execute without ssh */
 		cmdline = g_strconcat(cmd, NULL);
